IdentityCore/src/webview/systemWebview/ios/MSIDAuthenticationSession.m

```diff
@@ -139,14 +193,22 @@ - (void)startWithCompletionHandler:(MSIDWebUICompletionHandler)completionHandler
             _webAuthSession = [[ASWebAuthenticationSession alloc] initWithURL:_startURL
                                                             callbackURLScheme:_callbackURLScheme
                                                             completionHandler:authCompletion];
+            if (@available(iOS 13.0, *))
+            {
+                _webAuthSession.presentationContextProvider = self;
+                _webAuthSession.prefersEphemeralWebBrowserSession = self.prefersEphemeralWebBrowserSession;
+            }
+            
             if ([_webAuthSession start]) return;
         }
         else
         {
+#if !TARGET_OS_UIKITFORMAC
```

Comment:
nit: should this be around else block for clarity?

Reply:
done